### ISSUE TITLE
Error page on localhost even after installing everything

### ISSUE THREAD

**Issue by shubhscoder** [publiclab/mapknitter#397]
So, I just followed the steps mentioned in the readme to get mapknitter running. 
I installed everything mentioned and followed the procedure correctly to best of my knowledge. Then too I am getting the following error 
![mapknitter_error](https://user-images.githubusercontent.com/18489285/54223487-9eacbe80-451d-11e9-93ed-04d68735911c.png)

Also the server shows a internal error (status 500).
This is a part of the log
```
Rendered map/_authors.html.erb (965.8ms)
  Rendered maps/index.html.erb within layouts/application (1064.5ms)
Completed 500 Internal Server Error in 1276.1ms


Started GET "/" for 127.0.0.1 at 2019-03-12 23:18:28 +0530
Processing by MapsController#index as HTML
  Rendered layouts/_login_form.html.erb (0.8ms)
  Rendered maps/_search.html.erb (0.2ms)
  Rendered map/_regions.html.erb (0.1ms)
  Map Load (67.0ms)  SELECT `maps`.* FROM `maps` WHERE (password = "" AND archived = "false") ORDER BY maps.id DESC LIMIT 12
  Rendered map/_authors.html.erb (81.1ms)
  Rendered maps/index.html.erb within layouts/application (94.8ms)
Completed 500 Internal Server Error in 98.5ms

```

 The reason I am guessing for this to happen is maybe no availability of data in the database. I tried rake db:seed, but there are no seeds available I guess. 
Any solution would be very helpful!

Thank you so much!

**Comment by shubhscoder** [comment]
@SidharthBansal I tried many things to fix this, but to no avail. Do you have any idea to get this working?
Thank you so much!

**Comment by jywarren** [comment]
Hmm, can you attach more of your error logs? Shall we write some seed data? Thank you @shubhscoder !!! I'm sorry you're having trouble!

**Comment by gauravano** [comment]
I was able to run the app locally and many other contributors too, so it's not seed file. Could you check the existance of database in mysql @shubhscoder? Also, see if there's migration pending.

**Comment by sashadev-sky** [comment]
@shubhscoder I think your error may have been resolved in some recently merged PRs (see discussion in #419)

Could you please make sure your repo is up to date and then confirm if you are still receiving this error?

**Comment by sashadev-sky** [comment]
closing due to stale!
